lib/iris/tests/integration/test_pickle.py

```diff
@@ -31,35 +31,43 @@
     from iris.fileformats.grib.message import GribMessage
 
 
+class Common(object):
+    def pickle_cube(self, path):
+        # Ensure that data proxies are pickleable.
+        cube = iris.load(path)[0]
+        with self.temp_filename('.pkl') as filename:
+            with open(filename, 'wb') as f:
+                pickle.dump(cube, f)
+            with open(filename, 'r') as f:
```

Comment:
@cpelley Shouldn't this really be `rb` instead of `r` ...

Reply:
(done!)